specification/eventgrid/data-plane/Microsoft.KeyVault/stable/2018-01-01/KeyVault.json

```diff
@@ -312,6 +312,40 @@
           "type": "number"
         }
       }
+    },
+    "KeyVaultVaultAccessPolicyChangedEventData": {
```

Comment:
Just to confirm, the duplicated word "Vault" is expected?

Reply:
Yes, it is for consistency with the other events pattern (add KeyVault prefix to event name)
Event Names:
...
SecretNewVersion....
KeyNewVersion
**VaultAccessPolicyChanged..**

If you add KeyVault prefix:
KeyVaultSecretNewVersion....
KeyVaultKeyNewVersion...
**KeyVaultVaultAccessPolicyChanged...**